typescript-sdk/src/client-sdk/services/traces/service.ts

```
@@ -0,0 +1,75 @@
+  private config: InternalConfig;
+
+  constructor(config: InternalConfig) {
+    this.config = config;
+
+    /**
+     * Wraps the service in a tracing proxy via the decorator.
+     */
+    return createTracingProxy(
+      this as TracesService,
+      tracer,
+    );
```

Biome (2.1.2)

[error] The constructor should not return a value.

The constructor is here:

Returning a value from a constructor may confuse users of the class.

(lint/correctness/noConstructorReturn)